src/java-interop/java-interop-gc-bridge-mono.cc

```diff
@@ -24,6 +24,10 @@
 
 #include <dlfcn.h>
 
+#if defined (DYLIB_MONO)
```

Comment:
I'd rather us replace `DYLIB_MONO` with `XAMARIN_ANDROID` here and define the latter macro in Xamarin.Android - it's more specific and clear as to its purpose

Reply:
replaced with XAMARIN_ANDROID_DYLIB_MONO as discussed